whylog/config/rule.py

```diff
@@ -4,13 +4,28 @@
 import six
 
 from whylog.config.parsers import RegexParserFactory
+from whylog.constraints.verifier import Verifier
 
 
 class Rule(object):
     def __init__(self, causes, effect, constraints):
         self._causes = causes
```

Comment:
add `assert causes` above this line to prevent `_gather_causes_frequency_information` exploding at a random moment